mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.ml.regression
 
-import breeze.stats.distributions.{Gaussian => GD}
+import breeze.stats.distributions.{Gaussian => GD, StudentsT}
```

Comment:
We can import `distributions` as `dist` then use `dist.Abc` in the code. See comments below.